nix/sources.json

```diff
@@ -0,0 +1,32 @@
+{
+    "haskell.nix": {
+        "branch": "master",
+        "description": "Alternative Haskell Infrastructure for Nixpkgs",
+        "homepage": "https://input-output-hk.github.io/haskell.nix",
+        "owner": "input-output-hk",
+        "repo": "haskell.nix",
+        "rev": "af10907a11fd00e9ca7f108198200e7beedc0974",
+        "sha256": "171ikxsgvihqn8grpg8pz6g75fhmczsl0bfivhfaqvqynqpivinj",
+        "type": "tarball",
+        "url": "https://github.com/input-output-hk/haskell.nix/archive/af10907a11fd00e9ca7f108198200e7beedc0974.tar.gz",
+        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
+    },
+    "ligo": {
+        "ref": "dev",
+        "repo": "https://gitlab.com/ligolang/ligo",
+        "rev": "82b3d634c1ff81b611f2779ee382a8ee8adf3af9",
```

Comment:
Does it include https://gitlab.com/ligolang/ligo/-/merge_requests/585 (I am a bit lazy to check and you probably know the answer)?

Reply:
I don't know.

Updated just in case.